controllers/gatekeeper_controller.go

```diff
@@ -1038,7 +1039,31 @@ func openShiftDeploymentOverrides(obj *unstructured.Unstructured) error {
 
 	err = unstructured.SetNestedField(obj.Object, containers, "spec", "template", "spec", "containers")
 	if err != nil {
-		return errors.Wrapf(err, "Failed to set the OpenShift overrides")
+		return errors.Wrapf(err, "Failed to set --disable-cert-rotation for OpenShift")
+	}
+
+	const volumeErrMsg = "Failed to set the certificate volume mount for OpenShift"
+
+	volumes, _, err := unstructured.NestedSlice(obj.Object, "spec", "template", "spec", "volumes")
+	if err != nil || len(volumes) == 0 {
+		return errors.Wrapf(err, volumeErrMsg)
+	}
```

Comment:
So we're not creating the volume if it doesn't exist?

Reply:
Volumes come the cr from gatekeeper. So actually, it is 0 percent for this versions. Why is this added, maybe the next version could be was removed mistakenly. 
Should I remove?

Reply:
Ah, sorry I should have looked at the deployment first. I think it's fine to leave in then. In that case we'll figure out pretty quickly if it gets removed and we can address it.

Reply:
I added tests for our next version